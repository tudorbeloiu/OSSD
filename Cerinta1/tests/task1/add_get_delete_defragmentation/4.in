40
4
1
1
246
1070
3
246
1
9
122
3355
142
6254
124
940
113
8337
152
6048
95
3086
195
2184
219
5696
189
5654
1
1
85
5571
2
142
1
1
89
4418
2
189
4
2
122
3
142
3
95
2
124
4
3
195
3
122
2
189
2
219
3
219
3
85
1
9
80
6138
57
6727
235
3407
34
8855
208
2109
145
2545
198
975
16
1507
224
4576
1
5
25
4662
105
1550
85
5511
147
5054
97
6138
1
5
43
7354
73
2436
67
5843
183
195
9
233
3
235
4
2
113
1
3
245
6225
74
1219
22
918
1
6
46
6374
196
2286
136
5081
15
8596
223
783
96
740
2
74
2
80
4
2
96
1
4
206
3177
62
8970
58
6534
131
5611
3
74
4
2
97
2
96
3
198
2
105
1
5
185
4742
37
2368
60
6525
242
3843
201
7604
