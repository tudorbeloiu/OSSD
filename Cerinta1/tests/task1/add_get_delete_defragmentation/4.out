246: (0, 133)
122: (0, 419)
142: (0, 0)
124: (420, 537)
113: (0, 0)
152: (0, 0)
95: (538, 923)
195: (0, 0)
219: (0, 0)
189: (0, 0)
85: (0, 0)
(0, 0)
89: (0, 0)
(0, 0)
122: (0, 419)
124: (420, 537)
95: (538, 923)
(0, 419)
122: (0, 419)
124: (420, 537)
95: (538, 923)
122: (0, 419)
124: (420, 537)
(420, 537)
122: (0, 419)
124: (420, 537)
122: (0, 419)
124: (420, 537)
124: (420, 537)
(0, 0)
(0, 0)
124: (420, 537)
124: (420, 537)
80: (0, 0)
57: (0, 0)
235: (538, 963)
34: (0, 0)
208: (0, 263)
145: (0, 0)
198: (264, 385)
16: (0, 0)
224: (0, 0)
25: (0, 0)
105: (0, 0)
85: (0, 0)
147: (0, 0)
97: (0, 0)
43: (0, 0)
73: (0, 0)
67: (0, 0)
183: (386, 410)
9: (964, 993)
208: (0, 263)
198: (264, 385)
183: (386, 410)
124: (420, 537)
9: (964, 993)
208: (0, 263)
198: (264, 385)
183: (386, 410)
124: (411, 528)
9: (529, 558)
(0, 0)
245: (0, 0)
74: (559, 711)
22: (712, 826)
46: (0, 0)
196: (0, 0)
136: (0, 0)
15: (0, 0)
223: (827, 924)
96: (925, 1017)
(559, 711)
(0, 0)
208: (0, 263)
198: (264, 385)
183: (386, 410)
124: (411, 528)
9: (529, 558)
74: (559, 711)
22: (712, 826)
223: (827, 924)
96: (925, 1017)
(925, 1017)
206: (0, 0)
62: (0, 0)
58: (0, 0)
131: (0, 0)
208: (0, 263)
198: (264, 385)
183: (386, 410)
124: (411, 528)
9: (529, 558)
22: (712, 826)
223: (827, 924)
96: (925, 1017)
208: (0, 263)
198: (264, 385)
183: (386, 410)
124: (411, 528)
9: (529, 558)
22: (559, 673)
223: (674, 771)
96: (772, 864)
(0, 0)
(772, 864)
208: (0, 263)
183: (386, 410)
124: (411, 528)
9: (529, 558)
22: (559, 673)
223: (674, 771)
96: (772, 864)
(0, 0)
185: (0, 0)
37: (0, 0)
60: (0, 0)
242: (0, 0)
201: (0, 0)
